atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/ImportRenderer.java

```diff
@@ -25,7 +25,7 @@
 import com.google.common.collect.Iterables;
 
 public class ImportRenderer extends Renderer {
-    private static final List<String> IMPORT_PREFIXES = ImmutableList.of("java.", "javax.", "org.", "com.");
+    private static final List<String> IMPORT_PREFIXES = ImmutableList.of("java.", "javax.", "org.", "com.", "net.");
```

Comment:
For some reason, this slightly breaks the import rendering - `renderImports` should not add a blank line when there are no imports in the block (as is the case with `net.` in most files). However, extra blank lines are added.

Running `./gradlew generateSchemas` will expose this.